tendrl/ceph_integration/manager/tendrl_definitions_ceph.py

```diff
@@ -61,6 +46,22 @@
           run: tendrl.ceph_integration.objects.pool.atoms.delete.Delete
           type: Delete
           uuid: 9a2df258-9b24-4fd3-a66f-ee346e2e3720
+      flows:
+        DeletePool:
+          atoms:
+            - tendrl.ceph_integration.objects.pool.atoms.delete
+          description: "Delete Ceph Pool"
+          enabled: true
+          inputs:
+            mandatory:
+              - Pool.pool_id
+              - Tendrl_context.sds_name
+              - Tendrl_context.sds_version
+              - Tendrl_context.cluster_id
+          run: tendrl.ceph_integration.flows.delete_pool.DeletePool
```

Comment:
As we talked and discussed, as part of this PR the parsing logic for the flows would require changes as we should be able to figure out if a flow is generic one or specific to an object. Lets take that change as part of this PR.